openmicroscopy/openmicroscopy
# Date filter

Unify date displayed in the central panel and the search by date in main search
Also remove hard-coded url

To test:
 * Check that the date displayed in the filtering dialog (central panel) matches the display of the search component.
 * Go to the search tab, Click on the info button next to text field. 
 * Check that the dialog pops up and click on the url.

--no-rebase
